### ISSUE TITLE
Cascadia Codeバージョン

### ISSUE THREAD

**Issue by miiton** [miiton/Cica#55]
https://github.com/microsoft/cascadia-code

リガチャもいける...か？

**Comment by hanubeki** [comment]
Cascadia Codeは他の (Regularウェイトの) フォントと比べ少し太めなのでCascadia CodeにRounded Mgen+をマージするとしたらRegularではなくMediumにした方が違和感が少ないと思います。

**Comment by hanubeki** [comment]
あと、unitsPerEmが2048なのに対し文字幅が1200と0.5emではないようなのでそれに合わせて調整する必要もありそうです。

**Comment by miiton** [comment]
そうですね、別のフォントとくっつける際は大きさや太さの調整はお互いのフォントの設定値からざっくりやった後に目検で微調整していくので、そういった情報を予め頂けると進めやすいです。
ありがとうございます。
